### PR TITLE
[W4.3d][T17-A4] Emil Koh Tze Ping

### PR DESCRIPTION
**Pull request by LimeFallacie**

### CODE REVIEW

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -53,6 +65,25 @@ public int hashCode() {
...
     }
 
+    /**
+     * fullAddress[0] : BLOCK
+     * fullAddress[1] : STREET
+     * fullAddress[2] : UNIT
+     * fullAddress[3] : POSTALCODE
+     */
+
+    public String[] AddressSplitter(String address){
```
Comment (liuyang1213) [inline src/seedu/addressbook/data/person/Address.java]: This naming style is not allowed by our coding standard. Names representing methods must be _verbs_ and written in _camelCase_, as specified in [coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#naming-conventions). What do you think it should be?

File: src/seedu/addressbook/data/person/Block.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+    private static String blkNumber = "";
```
Comment (liuyang1213) [inline src/seedu/addressbook/data/person/Block.java]: `static` means all instances share the same value of this variable, should you use it here? Same for PostalCode, Street and Unit.

**Comment by liuyang1213** [comment]
@LimeFallacie  Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/data/person/Address.java (hunk 1 above)
Comment (LimeFallacie) [reply src/seedu/addressbook/data/person/Address.java]: splitAddress....? (oops)

**Comment by LimeFallacie** [comment]
amendments made to mistakes in LO. 
closing PR

**Pull request closed without merging**